components/MobileNavigation.js

```diff
@@ -20,23 +20,53 @@ export default class MobileNavigation extends Component {
 
   state = {
     showMobileNav: false,
-    expandMobileNav: false
+    expandMobileNav: false,
+    nowSelectMenu: null,
+    scrollPosition: null
   }
 
   componentDidMount () {
+    const { navs } = this.props
     window.addEventListener('scroll', this.handleScroll)
+    const scrollPosition = navs.reduce((obj, nav, index) => {
+      const navScroll = {
+        name: nav.label,
+        scroll: document.getElementById(nav.label.toLowerCase()).getBoundingClientRect().top + window.scrollY
```

Comment:
`document.getElementById(nav.label.toLowerCase())`

Do not depend on the label, because scrolling is based on `href`. Instead, use `document.getElementById(nav.href.slice(1))`.